data-prepper-plugins/grok-prepper/README.md

```diff
@@ -45,9 +45,13 @@ TODO: provide examples for using each configuration
 
 * `target_key` (Optional): A `String` that will wrap all captures for a Record in an additional outer key value. Default value is `null`
 
+
+* `timeout_millis` (Optional): An `int` that specifies the maximum amount of time, in milliseconds, that matching will be performed on an individual Record before it times out and moves on to the next Record.
+Setting a `timeout_millis = 0` will make it so that matching a Record never times out. Default value is `30,000`
```

Comment:
This could be simplified: "Setting a `timeout_millis = 0` will disable this feature. Default value is `30,000`"